tags/faq/rule8.ytag

```diff
@@ -0,0 +1,7 @@
+type: text
+
+---
+
+> :eight: Do not mention names from MCP, Mojang or other proprietary mappings, even if you use spoilers. 
+
+This is due to them having a more restrictive license than yarn. Many yarn contributors chat here and we don't want them to pick up names from other mappings due to their licensing. 
```

Comment:
I think you can freely mention whatever is available in the game jar (like constant values (strings, integers), a few blaze3d classes) as long as you can reasonably derive something without looking at mcp or mojang mappings